partcad/src/partcad/part.py

```
@@ -61,11 +47,12 @@ def clone(self):
     async def get_mcftt(self, property: str):
-        # Get the material, color, finish, texture or tolerance of the part
-        if (
-            self.vendor is None
-            and self.sku is None
-            and (not "parameters" in self.config or not property in self.config["parameters"])
+        """Get the material, color, finish, texture or tolerance of the part."""
+
+        store_data = self.get_store_data()
+
+        if (not store_data.vendor or not store_data.sku) and (
+            not "parameters" in self.config or not property in self.config["parameters"]
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)